cmd/common_konnect.go

```diff
@@ -246,7 +246,7 @@ func syncKonnect(ctx context.Context,
 		return err
 	}
 
-	stats, errs, _ := s.Solve(ctx, parallelism, dry, false)
+	stats, errs, _ := s.Solve(ctx, parallelism, dry, false, false)
```

Comment:
Style: the new `bool` parameter could get its own type (e.g. `type ApplyType bool`) and then a set of named constants so that we don't end up with call sites that just use verbatim `true` / `false` which are hard to reason about unless function signature is looked up.

Reply:
Agreed. I've pushed an update for this in https://github.com/Kong/deck/pull/1508/commits/152977482599301b8559d3c7f70801bcb5222a3a